pyproject.toml

```diff
@@ -256,10 +271,7 @@ target-version = "py39"
 
 [tool.ruff.lint]
 ignore = [
-  "ANN101", # Missing type annotation for `self` in method
-  "ANN102", # Missing type annotation for `cls` in class method
```

Comment:
These were removed in Ruff [0.8.0](https://github.com/astral-sh/ruff/releases/tag/0.8.0).